Rust/Rust.sublime-syntax

```diff
@@ -494,32 +548,15 @@ contexts:
           scope: punctuation.definition.generic.begin.rust
           pop: true
         - include: generic-angles-contents
-    - match: '\['
-      scope: punctuation.section.group.begin.rust
-      push:
-        - meta_scope: meta.group.rust
-        - match: '\]'
-          scope: punctuation.section.group.end.rust
-          pop: true
-        - match: ';'
-          scope: punctuation.separator.rust
-          set:
-            - match: '\]'
-              scope: punctuation.section.group.end.rust
-              pop: true
-            - include: constant-integer-expression
-            - match: '(?=\S)'
-              pop: true
-        - include: generic-angles-contents
     - include: type-any-identifier
-    - match: '{{support_type}}'
-      scope: support.type.rust
-    - match: ':'
+    - match: '{{identifier}}'
+    - match: ':|,'
       scope: punctuation.separator.rust
-    - match: '\+'
-      scope: keyword.operator.rust
-    - match: '\bas\b'
+    - match: '\+|='
       scope: keyword.operator.rust
+    - match: '(?=\S)'
+      scope: invalid.illegal.rust
```

Comment:
As `(?=\S)` is a lookahead which does not consume anything, the `scope` is never applied to something.